WearSpeakerSample/wear/src/main/java/com/example/android/wearable/speaker/SoundRecorder.kt

```diff
@@ -64,12 +63,6 @@ class SoundRecorder(
 
         val intSize = AudioTrack.getMinBufferSize(RECORDING_RATE, CHANNELS_OUT, FORMAT)
 
-        audioManager.setStreamVolume(
-            AudioManager.STREAM_MUSIC,
-            audioManager.getStreamMaxVolume(AudioManager.STREAM_MUSIC),
-            0 /* flags */,
-        )
```

Comment:
I removed this, since it forces device volume to max, which includes setting bluetooth headphone volume to max.